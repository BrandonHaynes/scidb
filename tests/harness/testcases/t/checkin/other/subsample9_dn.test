--setup
CREATE ARRAY dense <a: double > [x=0:5,3,0, y=0:5,3,0]
load(dense,'${TEST_DATA_DIR}/dense.txt')

--test
subarray(dense, 2, 0, 5, 5)

--cleanup
remove (dense)

