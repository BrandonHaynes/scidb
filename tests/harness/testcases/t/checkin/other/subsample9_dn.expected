Query was executed successfully

{x,y} a
{0,0} 11
{0,1} 12
{0,2} 13
{1,0} 21
{1,1} 22
{1,2} 23
{2,0} 31
{2,1} 32
{2,2} 33
{0,3} 14
{0,4} 15
{0,5} 16
{1,3} 24
{1,4} 25
{1,5} 26
{2,3} 34
{2,4} 35
{2,5} 36
{3,0} 41
{3,1} 42
{3,2} 43
{4,0} 51
{4,1} 52
{4,2} 53
{5,0} 61
{5,1} 62
{5,2} 63
{3,3} 44
{3,4} 45
{3,5} 46
{4,3} 54
{4,4} 55
{4,5} 56
{5,3} 64
{5,4} 65
{5,5} 66

{x,y} a
{0,0} 31
{0,1} 32
{0,2} 33
{1,0} 41
{1,1} 42
{1,2} 43
{2,0} 51
{2,1} 52
{2,2} 53
{0,3} 34
{0,4} 35
{0,5} 36
{1,3} 44
{1,4} 45
{1,5} 46
{2,3} 54
{2,4} 55
{2,5} 56
{3,0} 61
{3,1} 62
{3,2} 63
{3,3} 64
{3,4} 65
{3,5} 66

Query was executed successfully

